### PR TITLE
Separating example from usage syntax

### PR DESCRIPTION
**Pull request by animenon**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -57,6 +57,12 @@ integer.
                           picked randomly when encode.
     -e ENCODING, --encoding ENCODING
                           set encoding, only useful when base is not set
+                          
+
+For Example,
+
+::
```
Comment (McSinyx) [inline README.rst]: Could you change the **4** lines above to `For example::`?

**Review by McSinyx — CHANGES_REQUESTED** [review]
Thank you very much for the edit. The following request is simply for style consistancy. Also after the change, please rebase to one commit.

File: README.rst (hunk 1 above)
Comment (animenon) [reply README.rst]: `.rst` is quite new for me.

Comment (McSinyx) [reply README.rst]: It was new for me at the time I wrote this README too :stuck_out_tongue:

I think you miss my request to remove the trailing newline (L61) and rebase the 2 commits together. It should be as easy as `git rebase -i HEAD~2` and `git push --force`.

**Comment by McSinyx** [comment]
Merged. Thanks. Also I'll look again into this repo trynna refactor it and might need some help. If you're interested you might want to watch the repository.

**Pull request merged**